go/kbfs/libhttpserver/content_type_utils.go

```diff
@@ -76,10 +76,14 @@ func getGUIFileContext(contentTypeRaw, contentDispositionRaw string) (
 	viewType keybase1.GUIViewType, invariance string) {
 	contentTypeProcessed := beforeSemicolon(contentTypeRaw)
 	disposition := beforeSemicolon(contentDispositionRaw)
+
+	if disposition == "attachment" {
```

Comment:
Why not leave this in the `switch`, and let it return below like all the others?

Reply:
Those inside the switch are all about contentType but this is disposition, so I thought it'd be better to make this one explicit so it's less likely we break it.